doc/prepare_raspberrypi.rst

```diff
@@ -53,6 +70,66 @@ Log in to it::
 
 Enter the password mentioned just above.
 
+Installation using Windows
+--------------------------
+This section describes how to install the Raspberry Pi OS
+and how to enable ssh and serial console so that the Raspberry Pi can be 
+used in headless mode without a display and keyboard connected.
+
+Step 1. Write Raspberry Pi OS image to SD card using Rasberry Pi Imager.
+
+* Download and install Raspberry Pi Imager from 
+  https://www.raspberrypi.org/software/
+* Start Raspberry Pi Imager
+* In the Select OS dialog choose full version
+* Select SD-card
+* Press Write
+
+Step 2. Initial configuration of Raspberry Pi OS.
+
+* Eject SD-card 
+* Reinsert SD-card in windows PC. The SD-card will be shown as external drive named "boot".
+* Enable ssh by creating an empty file named ''ssh'' in the root folder of "boot". 
```

Comment:
Use double back-ticks for inline code examples. (The example four lines below works fine)

Reply:
Done, I think I need new glasses.